packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -478,12 +478,12 @@ A timeout of 3 seconds is enforced on all outbound requests, with one retry in t
 
 Note that this execution endpoint is not tied to any other functionality in Okta and should only be used for testing purposes.
```

Comment:
make it a note - if they go so far as to point out something to note - we should pull it out into a note, I think! Also, contractions for a more conversational tone, and present tense (avoid be verbs).

>**Note:** This execution endpoint isn't tied to any other functionality in Okta and you should only use it for testing purposes.